curator-recipes/src/main/java/org/apache/curator/framework/recipes/cache/TreeCacheEvent.java

```diff
@@ -69,6 +69,9 @@
          * While the connection is down, the TreeCache will continue to have its state from before it lost
          * the connection and after the connection is restored, the TreeCache will emit normal child events
          * for all of the adds, deletes and updates that happened during the time that it was disconnected.
+         * </p><p>
+         * After reconnection, the cache will resynchronize its internal state with the server, then fire a
+         * {@link #INITIALIZED} event.
          * </p>
          */
         CONNECTION_RECONNECTED,
```

Comment:
Shouldn't isInitialized.set(false) occur for SUSPENDED (i.e. any connection loss)?